data/pathfinder/paizo/roleplaying_game/core_rulebook/cr__datacontrol.lst

```diff
@@ -32,7 +32,7 @@ FACTSETDEF:DEITY|Race			DATAFORMAT:String		VISIBLE:YES
 
 
 
-#FACTDEF:CHECKS|MAINSTAT		DATAFORMAT:STRING	VISIBLE:YES	SELECTABLE:NO	REQUIRED:NO	DISPLAYNAME:MAINSTAT		EXPLANATION:Abbreviation of Stat that is used for the Check (Save)
+#FACTDEF:SAVES|MAINSTAT		DATAFORMAT:STRING	VISIBLE:YES	SELECTABLE:NO	REQUIRED:NO	DISPLAYNAME:MAINSTAT		EXPLANATION:Abbreviation of Stat that is used for the Check (Save)
```

Comment:
CHECK not CHECKS or SAVES
Sorry, but there is heritage here so we will have to work this part out later.